pkg/ebpf/telemetry/errors_telemetry.go

```diff
@@ -249,34 +249,54 @@ func (e *ebpfTelemetry) initializeHelperErrTelemetryMap(module names.ModuleName,
 	return nil
 }
 
+// PatchConstant replaces the value for the provided relocation entry in the bpf bytecode
+func PatchConstant(symbol string, p *ebpf.ProgramSpec, eBPFKey uint64) error {
```

Comment:
does this function really belong to the bpf telemetry pkg?

Reply:
It's primary use is here. This functionality is being exported to avoid code duplication in the verifier stats collector.